pypowsybl/network.py

```diff
@@ -5397,3 +5397,46 @@ def transform_list_to_str(entry: _Union[str, _List[str]]) -> str:
         return entry.replace(' ', '')
     else:
         raise _pp.PyPowsyblError("argument should be a list of str or a str")
+
+
+def create_coupling_device(network: Network, df: _DataFrame = None, raise_exception: bool = False,
+                           reporter: _Reporter = None, **kwargs: _ArrayLike) -> None:
+    """
+    Creates a coupling device on the network between two busbar sections of a same voltage level.
+
+    Args:
+        network: the network in which the busbar sections are.
+        df: Attributes as a dataframe.
+        raise_exception: an optional boolean indicating if an exception should be raised in case an error occurs during
+        computation.
+        reporter: an optional reporter to store the funtional logs.
+        kwargs: Attributes as keyword arguments.
+
+    Notes:
+        The voltage level containing the busbar sections must be described in node/breaker topology.
+        A closed breaker will be created as well as a closed disconnector on both given busbar sections to connect them.
+        If the topology extensions are present on the busbar sections then on every parallel busbar section, an open
+        disconnectors will be created to connect them to the breaker. If the two given busbar sections are the only two
+        parallel busbar sections, and they have the same section index then, only two closed disconnectors will be
+        created.
+
+        The input dataframe expects these attributes:
+
+        - **busbar_section_id_1**: the identifier of the busbars section on side 1
```

Comment:
about what we discuss it should be busbar section id or bus id shouldn't it ? and so maybe we should adapt the attribute name

Reply:
Actually this modification is not yet possible in bus/breaker, it will probably in next release but maybe for now it's better to leave it like that?